openadapt/models.py

```diff
@@ -314,3 +314,12 @@ class PerformanceStat(db.Base):
     start_time = sa.Column(sa.Integer)
     end_time = sa.Column(sa.Integer)
     window_id = sa.Column(sa.String)
+
+
+class MemoryStat(db.Base):
+    __tablename__ = "memory_stat"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    recording_timestamp = sa.Column(sa.Integer)
+    memory_usage = sa.Column(ForceFloat)
```

Comment:
@angelala3252 can you please rename this to something more descriptive, e.g. `memory_usage_kb` (or similar)?

Also, I noticed in the migration that the type does not match:

```
                    sa.Column('memory_usage', sa.Integer(), nullable=True),
                    sa.Column('timestamp', sa.Integer(), nullable=True),
```

And when I try to run `alembic upgrade head` I get:

```
KeyError: '3d3239ae4849'
```

This suggests to me that you created the revision off of a stale database state.

Can you please:

- delete openadapt.db
- check out main
- `alembic upgrade head`
- check out this branch
- `alembic revision ...`
- replace the current migration script with the new one

You may need to import the ForceFloat type.

Reply:
done in [fb290bb](https://github.com/MLDSAI/OpenAdapt/pull/270/commits/fb290bbadb2ee7d237611e61495a9497da5868e1) and [0d0c842](https://github.com/MLDSAI/OpenAdapt/pull/270/commits/0d0c8426b5fa5cfa57cadb0870e2133fac239683)